.github/workflows/publish.yml

```
@@ -0,0 +1,43 @@
+name: Publish Python 🐍 distributions 📦 to PyPI and TestPyPI
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  build-n-publish:
+    name: Build and publish Python 🐍 distributions 📦 to PyPI and TestPyPI
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@master
+      with:
+        fetch-depth: 0
+    - uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+
+    - name: Install python-build and twine
+      run: python -m pip install build twine
+
+    - name: Build package
+      run: python -m build --sdist --wheel .
+
+    - name: List result
+      run: ls -l dist
+
+    - name: Check long_description
+      run: python -m twine check dist/*
```

If you want the publish to not happen if there are warnings, it has to be strict.
```suggestion
      run: python -m twine check dist/* --strict
```

---

Currently there is warning about "long description type", not sure if we want to fail for this ? Oh, and I don't see `--strict` in `twine check`'s help ?

---

Hmm I see it here: https://twine.readthedocs.io/en/latest/#twine-check

---

Oh, funny:
```
❯ twine check --help
usage: twine check [-h] dist [dist ...]

positional arguments:
  dist        The distribution files to check, usually dist/*

optional arguments:
  -h, --help  show this help message and exit
```

---

Ah... Maybe pypa/twine#715 is not released yet. It is milestoned to 3.3.0 and current release is 3.2.0.